account_bank_match/data/account_bank_match_rule.xml

```diff
@@ -170,5 +170,17 @@
             <field name="type">bonus</field>
         </record>
 
+        <record
+            id="account_bank_match.account_bank_match_rule_18"
+            model="account.bank.match.rule">
+            <field name="active">True</field>
+            <field name="rule">[('invoice_partner_country_id', '=', '@_get_iban_country_id())@')]</field>

```

Comment:
invoice_partner_country_id is only added by ari custom module, so can not be used in 1200wd addon.

You need to use partner_invoice_id.country_id